sample-applications/AgentlessIdpSample/Controllers/ResumeController.cs

```diff
@@ -29,5 +32,17 @@ public IActionResult Index()
                 return View("~/Views/Error/Index.cshtml");
             }
         }
+        private static Boolean isAlphaNumeric(String referenceId)
+        {
+            Regex alphaNumeric = new Regex(@"^[a-zA-Z0-9]*$");
+            return alphaNumeric.IsMatch(referenceId);
+        }
+
+        private static Boolean checkRelativeUrl(String relativeUrl)
+        {
+            Regex resumePattern = new Regex(@"^(/idp/){1}[a-zA-Z0-9/]*(idp/){1}((SSO.ping)?(startSSO.ping)?){1}$");
```

Comment:
We probably should mention this specific validation in README.md, there are deployments out there that use URI rewriting and this validation may cause some deployment issues.

Reply:
hmm, I can just scrap the regex and keep the relative URI check to keep things simple, it's the more important check anyways for this.